tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,1023 @@
+<tool id="astronomical_archives" name="Astronomical Archives (IVOA)" version="0.9.0">
+    <description>queries astronomical archives through Virtual Observatory protocols</description>
+    <edam_operations>
+        <edam_operation>operation_0224</edam_operation>
+    </edam_operations>
+    <requirements>
+        <requirement type="package" version="5.2.2">astropy</requirement>
+        <requirement type="package" version="1.4.1">pyvo</requirement>
+    </requirements>
+    <command>
+      <![CDATA[
+
+        mkdir fits;
```

Comment:
would this work too?  else we need to change every command followed by something.
Actually as @francoismg will change this treatment to json, it will be easy to do &&

```suggestion
        mkdir fits || exit 1
```